roles/nameserver/tasks/config.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Create named data directory
+  file:
+    path: "{{ named_conf_data_dir }}"
+    state: directory
+
+- name: Create named.conf
+  template:
+    src: named.conf.j2
+    dest: "{{ named_conf_file }}"
+    validate: named-checkconf %s
+  notify: restart named
+  with_dict: "{{ named_domains }}"
+
+- name: Apply named daemon options
+  lineinfile:
+    dest: /etc/sysconfig/named
+    regexp: '^'
```

Comment:
Yep. I changed the way I wanted this to work in a recent change and forgot to re-add `OPTIONS`.  Will fix.